### ISSUE TITLE
ASV benchmarks have not run since mid-January on pandas.pydata.org/speed

### ISSUE THREAD

**Issue by qwhelan** [pandas-dev/pandas#25782]
If you go to http://pandas.pydata.org/speed/pandas/#regressions?sort=1&dir=desc or http://pandas.pydata.org/speed/pandas/#summarylist?sort=3&dir=desc and sort by descending date, you'll see no results more recent than 1/15/19. The same goes for all other benchmark suites also hosted on the speed site.

**Comment by jorisvandenbossche** [comment]
cc @TomAugspurger in case you have time to look at it

**Comment by TomAugspurger** [comment]
I don't know why, but the process created by airflow to run the benchmarks failed immediately. Restarting airflow seems to have solved it.

**Comment by qwhelan** [comment]
@TomAugspurger Results still seem to be stale, unfortunately.

**Comment by TomAugspurger** [comment]
I apparently broke something in a refactoring. The benchmarks are being run, but are now served from a different root, `http://pandas.pydata.org/speed/pandas/html/` (append the `/html`). Will figure that out later, unless someone wants to look at https://github.com/TomAugspurger/asv-collection or https://github.com/TomAugspurger/asv-runner before I get to it.

**Comment by TomAugspurger** [comment]
Should be fixed in https://github.com/asv-runner/asv-runner/commit/4396f99bd569194005a3b2ed56d8935a4a9e94a6. Turns out `rsync foo` is different from `rsync foo/` :/

As an aside, if anyone is interested in helping maintain the benchmark infrastructure, let me know. I've moved everything out of my personal account and into the asv-runner GitHub org.

Finally, I'm putting together a GitHub bot at https://github.com/asv-runner/asv-watcher to monitor the regression RSS feeds that asv generates, and post on PRs that may have caused regressions. Not quite as nice as the `@bot run benchmarks` bot @jni was hoping for, but it seems easier for now :)
